assets/scss/components/stencil/productView/_productView.scss

```
@@ -372,6 +372,8 @@
     border: $productOptions-list-border;
     list-style: none;
     margin: 0;
+    max-height: 400px;
+    overflow-y: scroll;
```

I think that scrollbar shouldn't be visible when items count less than height of dropdown.

`overflow-y: auto` - provides more flexible behavior 
```suggestion
    overflow-y: auto;
```

---

Thank you